libusb/hid.c

```diff
@@ -340,6 +345,88 @@ static int is_language_supported(libusb_device_handle *dev, uint16_t lang)
 	return 0;
 }
 
+static wchar_t *utf8_to_wchar(char *s)
```

Comment:
we already have a similar routine (using iconv) - would be great to have a common implementation if possible